lib/author.rb

```diff
@@ -0,0 +1,19 @@
+class Author
+  attr_accessor :first_name, :last_name
+
+  def initialize(first_name: nill, last_name: nill)
+    @id = Random.rand(1..1000)
+    @first_name = first_name
+    @last_name = last_name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
+    item.author = self
+  end
```

Comment:
Since you have to correct a pair of things, I ask you kindly to see if the implementation of your add_item method is correct with @Vgonma, I am not sure if this is ok.